microsoft/botframework-solutions
# [BotBuilder-Skills][TypeScript] Update with latest changes

## Description
Update with latest changes using `SkillHttpAdapter` and related classes.
## Testing Steps
1. Run `build.ts.ps1`
1. Check the packages generated in `./outputpackages`
